xbmc/platform/win32/XTimeUtils.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "utils/XTimeUtils.h"
+
+#include <FileAPI.h>
+
+namespace KODI
+{
+namespace TIME
+{
+
+void WINAPI Sleep(uint32_t dwMilliSeconds);
+{
+  Sleep(dwMilliSeconds);
```

Comment:
Use ::Sleep to make sure it doesn't call itself and to signal to the reader that it's meant to call a method in the global namespace. Same goes for the rest of the methods